use-cases/using-Intel-QAT-and-kata.md

```diff
@@ -97,24 +96,44 @@ chipset by executing the following.
 $ for i in 0434 0435 37c8 1f18 1f19; do lspci -d 8086:$i; done
 ```
 
-### Install necessary bundles for Clear Linux and disable kernel updates
+### Install necessary bundles for Clear Linux
 
-You need the Linux kernel headers to compile the out-of-tree kernel modules to
-support Intel QAT on the host. The `os-clr-on-clr` is a large development 
-bundle that you can install instead to include everything below with 
-additional items. This guide installs the Linux LTS kernel version `4.19` from
-2018 because the crypto modules are set to `m` and not `y`, which allows you
-to replace them with the out-of-tree driver. Otherwise you need to build your
-own Clear Linux kernel with the QAT drivers compiled in.
+Clear Linux version 30780 (Released August 13, 2019) includes a 
+`linux-firmware-qat` bundle that has the necessary QAT firmware along with a
+functional QAT host driver that can work with Kata Containers. Any version of
+Clear Linux after 30780 should work without having to replace the existing QAT
+host drivers. Other Linux distributions and previous versions of Clear Linux
+will require compiling and installing the QAT driver to the host. Those optional
+instructions are further below. In addition, the `os-clr-on-clr` is a large 
+development bundle that you can install instead to include everything below
+with additional items.  
 
 ```sh
 $ sudo swupd autoupdate --disable
-$ sudo swupd bundle-add network-basic kernel-lts2018 linux-lts-dev linux-dev make c-basic go-basic containers-virt dev-utils devpkg-elfutils devpkg-systemd
+$ sudo swupd bundle-add network-basic linux-firmware-qat linux-dev make c-basic go-basic containers-virt dev-utils devpkg-elfutils devpkg-systemd
+$ sudo clr-boot-manager update
 $ sudo systemctl start docker
 $ sudo systemctl enable docker
+$ sudo reboot
+```
+
+### Download Intel QAT drivers
+
+This will download the Intel QAT drivers from [`01.org`](https://01.org/intel-quickassist-technology). 
+Make sure to check the website for the latest version.
+
+```sh
+$ mkdir -p $QAT_SRC
+$ cd $QAT_SRC
+$ curl -O $QAT_DRIVER_URL
+$ QAT_DRIVER_NAME=$(echo $QAT_DRIVER_URL | rev | cut -d"/" -f1 | rev)
+$ tar xzvf $QAT_DRIVER_NAME
```

Comment:
`curl $QAT_DRIVER_URL | tar zx`

Reply:
This is happened to me.  I've had similar issues in the past with this type of pipe command.
```
eadams@otc-tme-tp-01a~/srcTEST/QAT $ curl -O $QAT_DRIVER_URL | tar xvz
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 4889k  100 4889k    0     0  12.1M      0 --:--:-- --:--:-- --:--:-- 12.1M
gzip: abort: EOF on input
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```
Same issue with wget
```
eadams@otc-tme-tp-01a~/srcTEST/QAT $ wget $QAT_DRIVER_URL | tar zx
--2019-09-10 14:46:36--  https://01.org/sites/default/files/downloads/qat1.7.l.4.6.0-00025.tar.gz
Resolving 01.org... 198.145.21.16
Connecting to 01.org|198.145.21.16|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 5007302 (4.8M) [application/x-gzip]
Saving to: ‘qat1.7.l.4.6.0-00025.tar.gz’

qat1.7.l.4.6.0-00025.tar.gz                100%[=====================================================================================>]   4.77M  23.3MB/s    in 0.2s

2019-09-10 14:46:37 (23.3 MB/s) - ‘qat1.7.l.4.6.0-00025.tar.gz’ saved [5007302/5007302]

gzip: abort: EOF on input
tar: Child returned status 1
tar: Error is not recoverable: exiting now
```

Reply:
drop `-O` from curl which takes the output to stdout.